pip/utils/__init__.py

```diff
@@ -105,28 +105,31 @@ def find_command(cmd, paths=None, pathext=None):
     # check if there are funny path extensions for executables, e.g. Windows
     if pathext is None:
         pathext = get_pathext()
-    pathext = [ext for ext in pathext.lower().split(os.pathsep) if len(ext)]
+    pathext = [ext for ext in pathext.lower().split(os.pathsep) if ext]
```

Comment:
What difference does this change make (other than a micro-optimisation)? ```ext``` is always a string, so testing ```ext``` or ```len(ext)``` are the same.